languages/js/src/Oso.test.ts

```diff
@@ -3,7 +3,7 @@ import { map, query } from '../test/helpers';
 
 test('#isAllowed', async () => {
   const o = new Oso();
-  o.loadStr('allow(1, 2, 3);');
+  await o.loadStr('allow(1, 2, 3);');
```

Comment:
was this hypothetically a race condition before? did the test ever fail?

Reply:
I'm not sure I've ever seen this particular one fail, but yeah unless Jest has some wild mechanism that subverts the JS event loop then this (and any of the other places in the test suite where we were missing `await`s) could've raced